internal/orchestrator/restore_workflow_ui_plan.go

```
@@ -0,0 +1,238 @@
+package orchestrator
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/tis24dev/proxsave/internal/logging"
+)
+
+func (w *restoreUIWorkflowRun) prepareBundleAndPlan() (bool, error) {
```

golangci-lint (2.12.1)

[error] func (*restoreUIWorkflowRun).prepareBundleAndPlan is unused

(unused)